uitools/common/CMakeLists.txt

```diff
@@ -0,0 +1,146 @@
+# Copyright 2012-2025 Esri
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+cmake_minimum_required(VERSION 3.10)
+
+set(ESRI_ARCGISRUNTIME_TOOLKIT_COMMON_SOURCES
```

Comment:
To simplify the code, `common/CMakeLists.txt` doesn't create a new lib, but just defines some variables. These variables are used in `tookitcpp.CMakeLists.txt` and `toolkitwidgets/CMakeLists.txt` to create the static libs of toolkit.